### ISSUE TITLE
Loading Error in the Book of Shadows

### ISSUE THREAD

**Issue by EnteringTheThicket** [MoriyaShiine/bewitchment#104]
I made the book of shadows but immediately got this when I opened it:

![Bewitchment Screenshot Loading Error](https://user-images.githubusercontent.com/82985802/115641083-607e4d80-a2e6-11eb-85ae-3df12d804743.png)

In addition, when I clicked on the Introduction page and one of the Chapters (Like Crops for example), I got these:

![Bewitchment Screenshot Chapters](https://user-images.githubusercontent.com/82985802/115641234-a4715280-a2e6-11eb-9935-1930001100f7.PNG)

This is my first time with this mod and while I'm no stranger to modding, I can still struggle with it. I'm just not sure if it's a compatibility error, a resource pack error or something else. I can still play the game thankfully but it's difficult when the book you need is having errors.

I will note that I am using an edited mod pack and I'm fine with uninstalling some mods if it will get Bewitchment to work. I think the mods are up to date besides a few resource packs. If you need the list, let me know! Thanks!

I provided the most recent log bellow.

-------------------------------------------------------------------------------

[Bewitchment Log.txt](https://github.com/MoriyaShiine/bewitchment/files/6354797/Bewitchment.Log.txt)

**Comment by Aelpecyem** [comment]
Have you tried reloading data, because this seems to be a patchouli related bug

**Comment by EnteringTheThicket** [comment]
> 
> 
> Have you tried reloading data, because this seems to be a patchouli related bug

Hi! I didn't know that was an option. I did some reloads but the same issue come up unfortunately. I typed in /reload, /reloadconfig and pressed F3 + T to see if that would fix it. I may be doing it wrong though! Any recommendations?
![image](https://user-images.githubusercontent.com/82985802/115720565-2eeb9d80-a34b-11eb-8702-7bb8279863bb.png)

**Comment by EnteringTheThicket** [comment]
Hey! Just an update. I looked into the /reload and restarted the game. I later saw that Kubejs was popping up with errors and warnings. I wasn't sure what happened until I read the Kubejs log (below) and realized that a lot of items were sharing the same recipe. I installed a different modpack that kept only the mods I wanted/needed and the Book of Shadows worked fine! I think the book was having difficulty loading the recipes since they "didn't exist". I'm unsure if it was the Kubejs mod in the pack or the other mods I was using. 

Sorry for the trouble! I'm excited to play the mod! Thanks!

[server.txt](https://github.com/MoriyaShiine/bewitchment/files/6360806/server.txt)
